cookbook/vis_mpl_seismic.py

```diff
@@ -0,0 +1,37 @@
+"""
+Seismic: seismic plotting utilities uses Obspy for SEGY reading
```

Comment:
@eusoubrasileiro could you change the title to something like "plotting a seismic section"? The title should be something that a person will search for. Just to be clear, most of my titles for recipes are awful and need to be changed.